app.py

```diff
@@ -3,18 +3,18 @@
 app = Flask(__name__)
 
 
-@app.route('/')
+@app.route('/videogames')
 def index():
     return "Main page for video game inclusion."
 
 
-@app.route('/<resource>', methods=['GET', 'POST'])
+@app.route('/videogames', methods=['GET', 'POST'])
 def info_pull(resource):
     assert resource == request.view_args['resource']
     return "Pull general information on video games that are being requested."
 
 
-@app.route('/<resource>/uuid', methods=['GET', 'PATCH', 'DELETE'])
+@app.route('/videogames/uuid', methods=['GET', 'PATCH', 'DELETE'])
```

Comment:
```suggestion
@app.route('/videogames/<uuid>', methods=['GET', 'PATCH', 'DELETE'])
```
uuid is variable, it's a unique ID per item. As is, this would only allow one static URL, `127.0.0.1:5000/videogames/uuid`  to respond.